client/Packages/com.beamable.server/Editor/Usam/Editor/MigrationHelper.cs

```diff
@@ -0,0 +1,57 @@
+using Beamable.Editor.UI.Model;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditorInternal;
+
+namespace Beamable.Server.Editor.Usam
+{
+	public static class MigrationHelper
+	{
+
+		public static List<string> GetDependentServices(string storageName)

```

Comment:
All methods from this file except from this one were copied from the editor popup that is used in the old microservice manager to show dependencies